src/apply-preserving-inline-style.js

```diff
@@ -222,11 +222,13 @@
 
   scope.apply = function(element, property, value) {
     ensureStyleIsPatched(element);
-    element.style._set(scope.propertyName(property), value);
+    if (typeof scope.propertyName == 'function') {
```

Comment:
`scope.propertyName` is an internal part of the polyfill, that it's missing in an older browser would indicate that something deeper is going wrong (we should track that down and fix that instead). I think not calling `apply` and `clear` in these cases would cause the polyfill to never apply any animations.

Reply:
Sorry, I never saw this comment. If you still want to debug this, I would try loading one of the development versions of the polyfill, e.g. `web-animations.dev.js` and stepping through the initialization logic.